compiler/build.gradle

```diff
@@ -0,0 +1,84 @@
+plugins {
+    id 'org.jetbrains.kotlin.jvm' version'1.3.61'
+    id 'com.google.protobuf' version '0.8.11'
+    id 'org.jetbrains.dokka' version '0.10.1'
+
+    // Generate IntelliJ IDEA's .idea & .iml project files
+    // Starting with 0.8.4 of protobuf-gradle-plugin, *.proto and the gen output files are added
+    // to IntelliJ as sources. It is no longer necessary to add them manually to the idea {} block
+    // to jump to definitions from Java and Kotlin files.
+    // For best results, install the Protobuf and Kotlin plugins for IntelliJ.
+    id 'idea'
+
+    // Provide convenience executables for trying out the examples.
+    id 'application'
+    id 'maven-publish'
+}
+
+description = 'The protoc plugin for gRPC Kotlin'
+mainClassName = 'io.grpc.kotlin.generator.GeneratorRunner'
+
+repositories {
+    google()
+    jcenter()
+    mavenCentral()
+    mavenLocal()
+}
+
+// Feel free to delete the comment at the next line. It is just for safely
+// updating the version in our release process.
```

Comment:
These comments are a bit strange to me. "Feel free to delete", but note that "it's just for safely ...".
Are we using the `CURRENT_GRPC_VERSION` tag as part of "our release process"?

Reply:
I removed that comment because it didn't make sense to me. And yes, I think keeping the `CURRENT_GRPC_VERSION` isa a good idea for consistency with `grpc-java` and to make it easier to find what needs updating when we bump versions.